src/ApiRequestPanelElement.d.ts

```diff
@@ -220,6 +220,11 @@ export declare class ApiRequestPanelElement extends EventsTargetMixin(LitElement
    * @attribute
    */
   allowCustomBaseUri: boolean;
+  /**
+   * List of credentials source
+   * @attribute
```

Comment:
This is an array so it shouldn't be an attribute. LitElement allows such a construct but it is not optimal. Please, remove the `@attribute` annotation.